### PR TITLE
Add question last cached time info

### PR DESCRIPTION
**Pull request by ranquild**
Epic https://github.com/metabase/metabase/issues/22826

How to test:
- Go to Admin -> Settings -> Caching
- Enable caching for question
- Go to a saved question and open the info panel
- Set caching time to e.g. 1 hour
- Refresh the question
- Refresh the question again after a minute. You should see the time it was cached

Screenshot 2022-07-08 at 13 48 21

### CODE REVIEW

File: frontend/src/metabase/lib/entities.js [hunk 1]
```diff
@@ -36,7 +36,7 @@
  *
  * Say we want to create a "books" entity, to be able to fetch a list of "books".
  *
- * Add the following line to `frontend/src/metabase/entities.index.js`:
+ * Add the following line to `frontend/src/metabase/entities.index.ts`:
```
Comment (WiNloSt) [inline frontend/src/metabase/lib/entities.js:39]: Why only change this line but not also line 43? But I'm not really sure this is deliberate.

File: frontend/src/metabase-lib/lib/Question.ts [hunk 2]
```diff
@@ -879,6 +879,10 @@ class QuestionInner {
     return this._card && this._card["last-edit-info"];
   }
 
+  lastQueryStart() {
+    return this._card && this._card["last_query_start"];
```
Comment (WiNloSt) [inline frontend/src/metabase-lib/lib/Question.ts:883]: Should we use optional chaining here? 
```suggestion
    return this._card?.["last_query_start"];
```

the only difference would be when `this._card` is **null**
- `this._card && this._card["last_query_start"]` would yield **null**
- while `this._card?.["last_query_start"]` would yield **undefined**

Side note: Just realize `last-edit-info` might be the only key that uses kebab case, should this be refactored eventually?

File: webpack.static-viz.config.js [hunk 3]
```diff
@@ -15,7 +15,7 @@ module.exports = {
 
   entry: {
     "lib-static-viz": {
-      import: "./static-viz/index.js",
+      import: "./static-viz/index.ts",
```
Comment (WiNloSt) [inline webpack.static-viz.config.js:18]: Not sure what's with this change. I don't see you renaming this file to `.ts` in this PR.

File: enterprise/frontend/src/metabase-enterprise/caching/index.js [hunk 4]
```diff
@@ -3,11 +3,11 @@ import { t, jt } from "ttag";
 import { hasPremiumFeature } from "metabase-enterprise/settings";
 import { PLUGIN_CACHING, PLUGIN_FORM_WIDGETS } from "metabase/plugins";
 import Link from "metabase/core/components/Link";
-import { CacheTTLField } from "./components/CacheTTLField";
-import { DatabaseCacheTTLField } from "./components/DatabaseCacheTTLField";
-import { QuestionCacheTTLField } from "./components/QuestionCacheTTLField";
-import { QuestionCacheSection } from "./components/QuestionCacheSection";
-import { DashboardCacheSection } from "./components/DashboardCacheSection";
+import CacheTTLField from "./components/CacheTTLField";
```
Comment (WiNloSt) [inline enterprise/frontend/src/metabase-enterprise/caching/index.js:6]: 👍

File: webpack.static-viz.config.js (hunk 3 above)
Comment (kulyk) [inline webpack.static-viz.config.js:18]: Is this change relevant to question caching?

File: enterprise/frontend/src/metabase-enterprise/caching/index.js [hunk 5]
```diff
@@ -3,11 +3,11 @@ import { t, jt } from "ttag";
...
+import DatabaseCacheTTLField from "./components/DatabaseCacheTTLField";
+import QuestionCacheTTLField from "./components/QuestionCacheTTLField";
+import QuestionCacheSection from "./components/QuestionCacheSection";
+import DashboardCacheSection from "./components/DashboardCacheSection";
```
Comment (kulyk) [inline enterprise/frontend/src/metabase-enterprise/caching/index.js:10]: Thank you!

**Review by WiNloSt — CHANGES_REQUESTED** [review]
I haven't run this branch, but it looks fairly simple. I have some comments on some questionable changes though. Also, you probably need to fix this 
image

**Review by kulyk — APPROVED** [review]

File: webpack.static-viz.config.js (hunk 3 above)
Comment (ranquild) [reply webpack.static-viz.config.js]: Nope, it's not intended, fixed

Comment (ranquild) [reply webpack.static-viz.config.js]: It's not intended, fixed

File: frontend/src/metabase-lib/lib/Question.ts (hunk 2 above)
Comment (ranquild) [reply frontend/src/metabase-lib/lib/Question.ts]: Fixed

File: frontend/src/metabase/lib/entities.js (hunk 1 above)
Comment (ranquild) [reply frontend/src/metabase/lib/entities.js]: It wasn't intended, fixed

**Comment by ranquild** [comment]
@WiNloSt Could you take a look? I cannot merge the PR since you've requested changes.

**Review by WiNloSt — APPROVED** [review]
Looks good to me 👍
